pkg/asset/manifests/powervs/cluster.go

```diff
@@ -171,6 +172,14 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 		},
 	}
 
+	if installConfig.Config.PowerVS != nil {
+		metadata, err := powervs.NewMetadata(installConfig.Config)
+		if err != nil {
+			return nil, err
+		}
+		installConfig.PowerVS = metadata
```

Comment:
The metadata should already have been initialized as part of the asset as comment above.

Reply:
No longer part of this PR.